criu/pie/restorer.c

```diff
@@ -328,10 +328,18 @@ static int restore_creds(struct thread_creds_args *args, int procfd,
 
 static inline int restore_pdeath_sig(struct thread_restore_args *ta)
 {
-	if (ta->pdeath_sig)
-		return sys_prctl(PR_SET_PDEATHSIG, ta->pdeath_sig, 0, 0, 0);
-	else
+	int ret;
+
+	if (!ta->pdeath_sig)
 		return 0;
+
+	ret = sys_prctl(PR_SET_PDEATHSIG, ta->pdeath_sig, 0, 0, 0);
+	if (ret) {
+		pr_err("Unable to set PR_SET_PDEATHSIG(%d): %d", ta->pdeath_sig, ret);
```

Comment:
please add `\n` at the end of the error message.

Reply:
Fixed. Thanks!